test/models/ReportingFrequencyViewModelSpec.scala

```diff
@@ -480,13 +510,123 @@ class ReportingFrequencyViewModelSpec extends UnitSpec with MockDateService with
         }
       }
     }
+
+    "exemptStatusCount" should {
+      "handle no exempt status" in {
```

Comment:
can this be rewritten to declare what the expected result should be so

something like

"return 0 for the exempt count and 3 for the non-exempt count"  

would possibly make it clearer to what the values in the tuples are at a glance